.gitignore

```diff
@@ -150,9 +150,6 @@ out
 # Keys
 *.pem
 
-# Stuff like keys and downloaded files
```

Comment:
Should we really remove this from the gitignore?

Reply:
I guess the idea is the resources directory may not be in the project directory any more, but if it is we don't want it added to git right?

Reply:
It might be better to leave this line for now to prevent the accidental addition of the resources directory to git.